### PR TITLE
fix: location output

### LINKED ISSUE #58: BP JSON Generator Error
The output of the BP.JSON generator is not validating as it has the wrong types for the location values .

They should be numbers not strings

image

### PR DESCRIPTION
**Pull request by JustinCast**
# Fix latitude & longitude type
Fix latitude & longitude type from string to number
## What does this PR do?
closes #58

### CODE REVIEW

**Review by tetogomez — APPROVED** [review]
LGTM

File: src/components/BPJsonGenerator/BPJsonGenerator.js [hunk 1]
```diff
@@ -429,19 +429,23 @@ const BPJsonForm = ({ accountName, bpJson, onSubmit }) => {
...
                 className={classes.formField}
               />
             </Grid>
             <Grid item xs={12} sm={6}>
               <TextField
                 onChange={(e) =>
-                  handleOnChange('longitude', e.target.value, 'location')
+                  handleOnChange(
+                    'longitude',
+                    Number(e.target.value),
```
Comment (adriexnet) [inline src/components/BPJsonGenerator/BPJsonGenerator.js:441]: @JustinCast you don't need to parse to Number the latitude field? it looks like this right now

```
onChange={(e) =>
  handleOnChange('latitude', e.target.value, 'location')
}
```

File: src/components/BPJsonGenerator/NodesForm.js [hunk 2]
```diff
@@ -176,6 +176,8 @@ const NodesForm = ({ nodes, nodeIndex, onSubmit, openModal, setOpenModal }) => {
     setCurrentNode(nodes[nodeIndex] || defaultNode)
   }, [nodes, nodeIndex])
 
+  console.log(nodes)
```
Comment (adriexnet) [inline src/components/BPJsonGenerator/NodesForm.js:179]: @JustinCast we should avoid commit `console.log` used for debug

Comment (JustinCast) [reply src/components/BPJsonGenerator/NodesForm.js:179]: Thanks

**Review by xavier506 — APPROVED** [review]

**Pull request merged**